example/pubspec.yaml

```diff
@@ -49,8 +49,8 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  assets:
-    - images/bg.png
+  # assets:
+  #  - images/bg.png
```

Comment:
This asset is used for RangePickerPageStyled.
Currently RangePickerPageStyled is not used in main example. But it stores solution for issues like this: https://github.com/MariaMelnik/flutter_date_pickers/issues/49.
Maybe I need to put it into main example.

Reply:
Flutter is complaining about not being able to find the asset. I don't like the solution at all. Will change it in a way it should be so that you can review the proposal.